src/test/java/com/softserve/academy/spaced/repetition/controller/UserControllerTest.java

```diff
@@ -46,6 +52,7 @@ public void setUp() {
     @Test
     public void testGetAuthorizedUserPublicInfo() throws Exception {
         when(userService.getAuthorizedUser()).thenReturn(getUser());
+        doNothing().when(publisher).publishEvent(new PointsBalanceEvent(this.getClass().getCanonicalName() , getUser()));
```

Comment:
This should be
`doNothing().when(publisher).publishEvent(Matchers.any(PointsBalanceEvent.class));`

Reply:
Done